SU2_CFD/src/driver_direct_multizone.cpp

```diff
@@ -638,44 +638,44 @@ bool CMultizoneDriver::Transfer_Data(unsigned short donorZone, unsigned short ta
   /*--- Select the transfer method according to the magnitudes being transferred ---*/
 
   if (transfer_types[donorZone][targetZone] == SLIDING_INTERFACE) {
-    transfer_container[donorZone][targetZone]->Broadcast_InterfaceData_Interpolate(solver_container[donorZone][INST_0][MESH_0][FLOW_SOL],solver_container[targetZone][INST_0][MESH_0][FLOW_SOL],
-                                                                                   geometry_container[donorZone][INST_0][MESH_0],geometry_container[targetZone][INST_0][MESH_0],
```

Comment:
In the interest of keeping the drivers independent of physics, would it make sense to make "transfer_types" a matrix of a little class (instead of enum) that stores also the donor/target solvers? You could then avoid all these if's.

Reply:
I agree this could be done better. I'll look into it soon and put a new PR request, probably with some other changes towards generalisation on the driver that are also needed.